patterns/vsi-extension/variables.tf

```diff
@@ -16,20 +16,19 @@ variable "prefix" {
 }
 
 variable "vpc_id" {
-  description = "The ID of the VPC where the VSI will be created."
+  description = "The ID of the VPC where you want to deploy the VSI. [Learn more](https://cloud.ibm.com/docs-draft/secure-infrastructure-vpc?topic=secure-infrastructure-vpc-ext-with-vsi)."
   type        = string
-  default     = null
 }
 
 variable "existing_ssh_key_name" {
-  description = "The ID of the VPC where the VSI will be created."
+  description = "The name of a public SSH key in the region where you want to deploy the VSI. [Learn more](https://cloud.ibm.com/docs/vpc?topic=vpc-ssh-keys). To create an SSH key, use the 'ssh_public_key' input instead."
```

Comment:
I don't think we should say "To create an SSH key" - We are not creating one, we are adding an SSH key value to the VPC region. @SirSpidey feel free to suggest rewording here